### PR TITLE
a more concise version of code fixing the URL error bug

### PR DESCRIPTION
**Pull request by truthgoddess**
@stevekrouse This is the more concise version of #559. This can be merged and #559 close. Can probably make it more concise, but not sure that is as necessary as what I already did. Thoughts?

### CODE REVIEW

**Comment by stevekrouse** [comment]
**[This is an automated integration to preview this pull request's changes to the website.]**

https://cdn.rawgit.com/stevekrouse/WoofJS/6df3cbf402cc155e4d94aa620f4e54607d68f3f6/index.html

**Comment by stevekrouse** [comment]
Much better! I took a shot at improving it a bit further myself: https://gist.github.com/stevekrouse/d0e0443f2269933ac4e0fe7321876a78

Note the anonymous function `t => code.includes(t)` which is a bit more concise, readable, modern way to do that.

Also note that I handled one other edge case and put a link to its reproduction (not strictly necessary but sometimes nice)\

Note: I didn't test my code so make sure you do that if you copy any of it into yours

File: create.html [hunk 1]
```diff
@@ -1560,7 +1560,16 @@ <h4 class="modal-body text-danger" :style="{display: !getID() ? 'block' : 'none'
         document.getElementById('preview').contentWindow.addEventListener("error", function(event) {
           var error = event.error
           if (error.type == "ImageLoadError") {
-            app.error = {message: "There is a problem with your image URL: " + error.url, lineno: code.substring(0, code.indexOf(error.url)).split('\n').length}
+              if (error.url === "") {
+                function findErrorType(type){
+                return (code.indexOf(type) != -1)
```
Comment (stevekrouse) [inline create.html]: your formatting here is off
```suggestion
                  return (code.indexOf(type) != -1)
```

**Comment by truthgoddess** [comment]
@stevekrouse I updated the branch to include your edits, and I think I formatting things correctly. Thoughts? If it looks good, it's ready to merge.
